doc/radosgw/lua-scripting.rst

```diff
@@ -297,6 +302,23 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Background Context
+--------------------
+The background context may be used for various need such as analytics, monitoring and
+caching data from other context executions.
+
+The ``RGW`` Lua table which is accesible from any context saves the data written into it
+while execution and this data could be read and used later in any other execution.
```

Comment:
The \`\`RGW\`\` Lua table is accessible from all contexts and saves data written to it
during execution so that it may be read and used later during other executions.